src/theory/sets/theory_sets_private.cpp

```diff
@@ -1032,739 +773,33 @@ void TheorySetsPrivate::checkDisequalities( std::vector< Node >& lemmas ) {
   }
 }
 
-void TheorySetsPrivate::checkCardBuildGraph( std::vector< Node >& lemmas ) {
```

Comment:
this function is renamed into CardinalityExtension::checkRegister

Reply:
Yes, that's the better name for it after the refactor.